Android/app/build.gradle

```diff
@@ -24,14 +24,14 @@ android {
             storePassword keystoreProperties['storePassword']
         }
     }
-    compileSdkVersion 32
-    buildToolsVersion '32.0.0'
+    compileSdkVersion 33
+    buildToolsVersion '33.0.0'
 
     defaultConfig {
         applicationId "app.intra"
         // Firebase Crashlytics requires SDK version 16.
         minSdkVersion 16
```

Comment:
*question* shall we update `minSdkVersion` to `23` as well (Outline Client is doing that)?

Reply:
I don't know. Can you investigate?